src/reducers/notebook-reducer.js

```diff
@@ -55,6 +55,9 @@ const notebookReducer = (state = newNotebook(), action) => {
     case 'TOGGLE_WRAP_IN_EDITORS':
       return Object.assign({}, state, { wrapEditors: !state.wrapEditors })
 
+    case 'UPDATE_NOTEBOOK_INFO':
```

Comment:
I think `IMPORT_NOTEBOOK` already handles the notebookInfo...  Would it be less redundant to just make sure `IMPORT_NOTEBOOK` is fired when making a new notebook, rather than adding a new action?

Reply:
We will be deprecating `IMPORT_NOTEBOOK`, iirc (@bcolloran has more to say about this I think) - in any case I can remove the `getNotebookInfo` call from it so we don't have this redundancy

Reply:
Ok.  Sure thing.

Reply:
ya importNotebook and IMPORT_NOTENOOK are kind of legacy -- they're meant to handle e.g. loading from localStorage and the URL trick, which may end up on the chopping block. And importNotebook is only called from within importInitialJsmd; this action+reducer will give us a way to update the clients knowledge about the notebook at any time, not just at load time.